playwright_stealth/js/chrome.app.js

```diff
@@ -16,60 +16,62 @@ window.chrome = {
 
 // That means we're running headful and don't need to mock anything
 if ("app" in window.chrome) {
-  return; // Nothing to do here
-}
-
-const makeError = {
-  ErrorInInvocation: (fn) => {
-    const err = new TypeError(`Error in invocation of app.${fn}()`);
-    return utils.stripErrorWithAnchor(err, `at ${fn} (eval at <anonymous>`);
-  },
-};
-
-// There's a some static data in that property which doesn't seem to change,
-// we should periodically check for updates: `JSON.stringify(window.app, null, 2)`
-const STATIC_DATA = JSON.parse(
-  `
-  {
-  "isInstalled": false,
-  "InstallState": {
-  "DISABLED": "disabled",
-  "INSTALLED": "installed",
-  "NOT_INSTALLED": "not_installed"
-  },
-  "RunningState": {
-  "CANNOT_RUN": "cannot_run",
-  "READY_TO_RUN": "ready_to_run",
-  "RUNNING": "running"
-  }
-  }
-      `.trim()
-);
-
-window.chrome.app = {
-  ...STATIC_DATA,
+  // return; // Nothing to do here
```

Comment:
Whats up with these commented out returns? We cant just comment out return. Essentially by doing this we disable "if" check. If thats our intention - instead of commenting out return, that check needs to be removed/reworked. And it would be helpful to provide rationale for such rework as I bet those checks are there for a reason.

Honestly, this PR is massive to be reviewed in one go. Very hard to parse it.
Is there a way to break it down?

Reply:
I forgot to remove these commented `return` statements. Initially, these return statements causes the rest of the script to not run. So, I wrapped it in an if else statement instead. I'll try to split the PR tomorrow but it's quite hard to because the scripts are linked together.